ConanInstall.cmake

```diff
@@ -4,11 +4,11 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   # Download automatically, you can also just copy the conan.cmake file
   # Put it in CMAKE_BINARY_DIR so we don't end up with two when building OpenStudioApplication
   if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+    # TODO: temporarily use my fork of conan.cmake (Output in stdout/stderr in real time + support MSVC 2019 in prevision)
     # message(STATUS "openstudio: Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
     # file(DOWNLOAD "https://github.com/conan-io/cmake-conan/raw/v0.13/conan.cmake"
```

Comment:
@jmarrec @lefticus what version of conan should we be using, I downgraded to 1.12.3 based on instructions in the wiki

Reply:
I use the one released a few hours ago, 1.14.4